.github/workflows/build.yaml

```diff
@@ -37,6 +37,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: true
+      - name: Check License
+        uses: apache/skywalking-eyes@main
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Should we officially merge this after the eye 0.1.0 release?

Reply:
Yes, this can be done after license-eye is released, but now just to verify it works